modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/NavigationButtons.js

```diff
@@ -33,10 +33,7 @@ const NavigationButtons = ({moveController, position}) => {
 			<div className="box left" onClick={() => moveController('left')}>
 				<ClayIcon className="icon" symbol="angle-down" />
 			</div>
-			<div
-				className="bottom box"
-				onClick={() => moveController('bottom')}
-			>
+			<div className="bottom box" onClick={() => moveController('down')}>
 				<ClayIcon className="icon" symbol="angle-down" />
 			</div>
```

Comment:
Empty line missing between 35 and 36. Is there a better semantic element instead these `div`s here?